catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -662,26 +662,42 @@ public Serializable parseDate(Serializable value) {
       return null;
     }
 
-    if (StringUtils.isEmpty((String) value)) {
+    String string = String.valueOf(value);
+
+    if (StringUtils.isBlank(string)) {
       return null;
     }
 
-    String string = String.valueOf(value);
+    if (StringUtils.isNumeric(string)) {
+      try {
+        return Instant.ofEpochMilli(Long.parseLong(string));
+      } catch (NumberFormatException ex) {
+        LOGGER.debug("Failed to create Epoch time from a numeric: {}", string, ex);
+        return null;
+      }
+    }
+
+    if (iso8601Z.matcher(string).matches()) {
+      return Instant.parse(string);
+    }
+
+    if (iso8601Offset.matcher(string).matches()) {
+      return OffsetDateTime.parse(string).toInstant();
```

Comment:
In other places I've been using the `toInstant(ZoneOffset.UTC)`

Reply:
For OffsetDateTime?  It doesn't seem to have that option.  Doesn't make much sense anyway I think.  An instant is not aware of a timezone, it is simply milliseconds since epoch.  Timezone is a representation, which OffsetDateTime should already be aware of.